tasks/cluster-start-and-reload.yml

```diff
@@ -0,0 +1,114 @@
+# Some changes to corosync.conf can be applied when a cluster is running by
+# instructing Corosync to reload its configuration. Other changes, however,
+# require the whole cluster to be restarted. Different Corosync versions have
+# different abilities as of which changes require restart and which are applied
+# fine by reloading.
+#
+# Currently, there is no reliable way to detect if a restart is necessary or we
+# would be fine with just the reload. Until then, we always restart to make
+# sure all changes are applied correctly and corner cases are avoided.
+#
+# The plan is to implement a functionality to pcs (a CLI or library command) to
+# inspect changes done to corosync.conf and decide if cluster restart is
+# necessary based on Corosync version.
+#
+# Corosync-qdevice does not support reload, it must always be restarted to
+# apply changes. If qdevice is not to be used in a cluster, we make sure it is
+# stopped. The only exception to that is when qdevice is not installed at all.
+
+- name: Get services status - detect corosync-qdevice
+  service_facts:
+
+- name: Stop cluster daemons to reload configuration
+  service:
+    name: "{{ item }}"
+    state: stopped
+  loop:
+    - pacemaker
+    - corosync-qdevice
+    - corosync
+  when:
+    - >
+        __ha_cluster_distribute_corosync_conf.changed
+        or __ha_cluster_distribute_corosync_authkey.changed
+        or __ha_cluster_distribute_pacemaker_authkey.changed
```

Comment:
ok - then you can add `'503'` to .ansible-lint to suppress the error

Reply:
fixed in 53ff3e8